test/unit/user-agent-service/test-visited.js

```diff
@@ -21,7 +21,7 @@ const { DB_ADD, TEMPIDS } = helpers;
 const perfTesting = false;
 
 describe('Storage.recentlyStarred', () => {
-  it('Stars and unstars.', async () => {
+  it('Stars and unstars.', async function() {
```

Comment:
It seems like it's a bug in the rule if this is needed. Fine to take it like this but is it worth checking if it is reported upstream?